### PR TITLE
Split reserved usernames list

### PR DESCRIPTION
**Pull request by kkampardi**
- Loop through all the loaded modules using module_loader
- Check for each module's reserved_usernames  attribute
- Check the username against the reserved_usernames

### CODE REVIEW

**Comment by SunilMohanAdapa** [comment]
@jvalleroy , I assume you are reviewing this.  Let me know if I should take it up.

File: plinth/modules/restore/__init__.py [hunk 1]
```diff
@@ -49,6 +49,8 @@
       '<a href=\'/restore/\'>reStore web-interface</a>.')
 ]
 
+reserved_usernames = ['node-restores']
```
Comment (jvalleroy) [inline plinth/modules/restore/__init__.py]: This should be 'node-restore'.

File: plinth/modules/repro/__init__.py [hunk 2]
```diff
@@ -28,7 +28,6 @@
 from plinth import frontpage
 from plinth import service as service_module
 from plinth.views import ServiceView
-
```
Comment (jvalleroy) [inline plinth/modules/repro/__init__.py]: Let's keep the extra line here, to separate imports from code.

**Comment by kkampardi** [comment]
I've just updated the fixes  for my typos @jvalleroy 
Can you please review?

**Comment by jvalleroy** [comment]
Thanks @kkampardi! I have merged this (after rebase to master).

**Pull request closed without merging**